STYLEGUIDE.md

```diff
@@ -18,7 +18,7 @@
 
 - Avoid single letter variables. Try using `idx` / `jdx` instead of `i` / `j` in for loops.
 
-- Prefer `urlString` over `URLString` (acronym prefix), `baseUrlString` over `baseURLString` (acronym infix), and `stripeId` over `stripeID` (acronym suffix)
```

Comment:
I'm not sure where this came from. It doesn't match the code. I took a shot at re-writing it to what I believe is the dominant style of the code base, but feedback on what the replacement should say would be great.